stable/datadog/templates/daemonset.yaml

```
@@ -60,9 +60,7 @@ spec:
       {{- end }}
       containers:
         {{- include "container-agent" . | nindent 8 }}
-        {{- if .Values.datadog.apm.enabled }}
-          {{- include "container-trace-agent" . | nindent 8 }}
-        {{- end }}
+        {{- include "container-trace-agent" . | nindent 8 }}
```

link to my global comment. it should be possible to desactivate the "trace-agent" deployment
```suggestion
        {{- if .Values.datadog.apm.enabled }}
        {{- include "container-trace-agent" . | nindent 8 }}
        {{- end }}
```